### PR TITLE
Made the interfaces for the Repositories

### PR DESCRIPTION
**Pull request by dannydjg16**
Made a few interface classes for the repositories, you guys can accept or deny but these should have all of the functions that we need and they're pretty separated by file

### CODE REVIEW

File: YourEpic/YourEpic.Domain/Interfaces/ICommentRepository.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+        public IEnumerable<Comment> GetComments(int epicID);
+
+        /// <summary>
+        /// Not sure about this one, dont know where we would display it is a response rather than a general comment
+        ///  the commentID parameter is the id of the comment that is being responded to
+        /// </summary>
+        /// <param name="commentID"></param>
+        /// <param name="">comment</param>
+        public bool RespondToComment(int commentID, Comment comment);
```
Comment (ghost) [inline YourEpic/YourEpic.Domain/Interfaces/ICommentRepository.cs:21]: I think to implement responding to comments we'd need a new table with commenter id and responder id and also the date so we can display them in order of date.

**Review by ghost — APPROVED** [review]
Besides the comment bit this all looks good. I like that you separated the functions between readers and publishers. This looks good though, we can decide later if we want to expand the comments.

**Review by alexsoto24 — APPROVED** [review]
Looks Good!

**Comment by dannydjg16** [comment]
Adding Interfaces to Domain for the Data Access Repos

**Pull request merged**